README.adoc

```diff
@@ -167,6 +167,35 @@ oc get routes
 
 NOTE: make sure to use `https` with the hostname/port you get from the command above, otherwise you'll see a message like: "Application is not available".
 
+== Auto injection of Jaeger Agent sidecars
+
+The operator can also inject Jaeger Agent sidecars in `Deployment` workloads, provided that the deployment has the annotation `inject-jaeger-agent` with a suitable value. The values can be either `"true"` (as string), or the Jaeger instance name, as returned by `kubectl get jaegers`. When `"true"` is used, there should be exactly *one* Jaeger instance for the same namespace as the deployment, otherwise, the operator can't figure out automatically which Jaeger instance to use.
```

Comment:
What happens if the there are more jaeger instances and inject-jaeger-agent=true? Does the deployment fail or it does not inject?

Reply:
Then nothing happens, unless there's a Jaeger instance named "true" (which I'm not even sure it's a valid name). :)